diff --git a/dmenu.c b/dmenu.c
index ccf55b5..caa4895 100644
--- a/dmenu.c
+++ b/dmenu.c
@@ -606,6 +606,7 @@ insert:
 			return;
 		/* fallthrough */
 	case XK_Up:
+	case XK_ISO_Left_Tab:
 		if (sel && sel->left && (sel = sel->left)->right == curr) {
 			curr = prev;
 			calcoffsets();
@@ -641,13 +642,13 @@ insert:
 		if (lines > 0)
 			return;
 		/* fallthrough */
-	case XK_Down:
+	case XK_Tab:
 		if (sel && sel->right && (sel = sel->right) == next) {
 			curr = next;
 			calcoffsets();
 		}
 		break;
-	case XK_Tab:
+	case XK_Down:
 		if (!sel)
 			return;
 		strncpy(text, sel->text, sizeof text - 1);
